package.json

```diff
@@ -31,13 +31,15 @@
   "devDependencies": {
     "@babel/core": "7.6.2",
     "@babel/polyfill": "^7.10.4",
+    "@material-ui/core": "4.11.0",
     "@testing-library/react": "^10.2.1",
     "autoprefixer": "9.6.0",
     "babel-jest": "24.9.0",
     "babel-loader": "^8.1.0",
     "babel-preset-cozy-app": "^1.9.0",
     "bundlesize": "0.18.0",
     "cozy-authentication": "1.19.1",
+    "cozy-doctypes": "1.74.7",
```

Comment:
are we using them?

Reply:
No, I don't know it ended up here, I removed it, thanks.